code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -841,6 +841,26 @@
 	taste_description = "rancid fungus"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
+/datum/reagent/consumable/moltobeso
+	name = "Molt'Obeso" //pardon my Italian
+	description = "Concentrated gluttony."
+	color = "#f8fc36"
+	taste_description = "gluttony"
+	nutriment_factor = 0 //the essence of this sauce is to stimulate hunger and improve the absorption of calories from food eaten
+	metabolization_rate = 0.25 * REAGENTS_METABOLISM
```

Comment:
This should've a lower rate, so it won't take more than 2 or 3 units to have an effect lasting a minute or more.

Reply:
made it 0.025 so it takes (according to my calculations) 1~ minute to metabolize around 3 units